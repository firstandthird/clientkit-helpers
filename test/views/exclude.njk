{% set filtered = items | exclude(exclude, 'slug') %}{% for filter in filtered %}{{ filter.name }}{% endfor %}
