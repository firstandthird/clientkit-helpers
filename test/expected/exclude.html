test 1test 3
